js/exercise-02.js

```diff
@@ -0,0 +1,12 @@
+/* 
+    1. Declare a variable using let with any name you like, but without a value.
+    2. On a separate line, assign the variable the String: "Moo!"
+    3. Log out this variable to the terminal using console.log
+    
+    Repeat the above steps but try using a const variable. Comment this new part of code and explain what is happening.
```

Comment:
This requirement is not explained in this file.

Have you tried to execute it? Does it work? Why

FIX: add a comment at the end of the file to explain this requirement.

Reply:
FIX: add a comment at the end of the file to explain this requirement.